docs/whirlpool/docs/01-Welcome.mdx

```diff
@@ -15,29 +15,41 @@ Whirlpools is an open-sourced concentrated liquidity automated market maker (CLA
   description: 'https://github.com/orca-so/whirlpools'
 }} />
 
-## Introduction to Solana.
+## Overview of Orca Whirlpools SDK suite
 
-On Solana, all data is stored in accounts, which operate like a key-value store. Each account has an address (the key) and holds specific data (the value). Some accounts store only data, like token accounts for holding tokens. Other accounts store programs that define the logic for interacting with the network. For example, the Token Program manages token accounts and facilitates transfers, so each time you send a token, you're actually sending a transaction to the Token Program, which then updates the balances involved.
+Orca provides a range of SDKs that cater to different levels of development needs for interacting with the Whirlpool Program on Solana and Eclipse. Whether you are managing liquidity, building applications that require pool infrastructure, or building automation tools that interact with the program, our SDKs cover a spectrum of functionality from low-level granular control to high-level abstractions.
 
-Similarly, the Whirlpools Program is a program that enables more advanced features like concentrated liquidity pools and swapping. It creates specific data accounts on your behalf, such as Whirlpool and Position accounts, which are essential for managing liquidity. While the Whirlpools Program "owns" these accounts, meaning it controls the logic governing them, you still have authority over specific aspects, like managing liquidity in a position through a corresponding Position NFT.
+What follows is a brief overview of our SDK suite, distinguishing between the Whirlpools SDKs and the Legacy SDK, and explaining their intended purposes and relationships.
 
-With the Whirlpools SDK, you can interact with these accounts, perform swaps, and manage positions with ease. And since Solana network transaction costs are low, you can execute these operations efficiently.
+### Whirlpools SDKs
 
-## Whirlpools SDK
+![Overview of Orca Whirlpools SDK suite](../static/img/01-Welcome/orca-sdks-overview.png)
 
-The Whirlpools SDK is an open-source library that allows developers to interact with the Whirlpools Program and is intended to be used in conjunction with the Solana SDK. Simply put, you use the Whirlpools SDK to create instructions, and the Solana SDK to send them to the network.
+The Whirlpools SDKs are our primary set of SDKs designed to provide enhanced, modular interaction with the Whirlpool Program. This offering is divided into three main components:
 
-Some of the main features are:
-- Create instructions to set up a new pool, open positions, add or remove liquidity, and swap tokens.
-- Get quotes for swaps.
-- Query the state of pools to retrieve information about asset prices, liquidity, and fees.
+#### 1. High-Level SDK
+- **TypeScript**: [@orca-so/whirlpools](https://www.npmjs.com/package/@orca-so/whirlpools) (Requires Solana Web3.js SDK ≥v2.0)
+- **Rust**: orca_whirlpools (in development)
```

Comment:
This will most likely only work with a single solana version

Reply:
Succesfully tried creating an `rpc` and call `fetchAllWhirlpoolWithFilter` with v2.0.0-rc.1, v2.0.0-rc-3, v2.0.0-rc.4. Does that still count as a single Solana version, namely 2.0.0?

Reply:
I meant the rust sdk. Will only work with `solana-(program|client|sdk)` v1.18